### PR TITLE
Add Arbitrum Support to Defi Kit with Cow Protocol Integration

### PR DESCRIPTION
**Pull request by jfrancai**
### What
This PR introduces Arbitrum support in the Defi Kit, starting with the integration of  Cow Protocol.

### Why
Arbitrum is a leading Layer 2 solution for DeFi, and Hopper plans to deploy its first vault on this network. 

### Implementation Details
- Updated contractAddressOverrides: Included the CowSwap order signer contract address.
- Added WETH address for Arbitrum.
- Refactored swap function: Modified the swap function to handle Arbitrum.
- Exposed arb1 in the SDK Package: Made Arbitrum network functionalities accessible through the SDK.

### CODE REVIEW

File: sdk/src/protocols/cowswap/index.ts [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
 
 const GPv2VaultRelayer = "0xC92E8bdf79f0507f65a392b0ab4667716BFE0110"
 const E_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
 
-const swap = async (options: {
-  sell: (`0x${string}` | "ETH")[]
-  buy?: (`0x${string}` | "ETH")[]
-  feeAmountBp?: number
-}) => {
+const getWethAddress = (chain: Chain) => {
```
Comment (sideris) [inline sdk/src/protocols/cowswap/index.ts:10]: please add return type

Comment (sideris) [reply sdk/src/protocols/cowswap/index.ts:10]: It should probably live somewhere else too. Not sure exactly where.

Maybe under something like `utils/getWethAddressByChain`

Comment (sideris) [reply sdk/src/protocols/cowswap/index.ts:10]: Change signature to `getWethAddressByChain`

**Review by sideris — CHANGES_REQUESTED** [review]

**Comment by sideris** [comment]
From structural perspective it looks great to me. 
Up to Nico or Jan to evaluate logic and approve or not. 

We also need to check Vercel issue.

**Review by nledo — APPROVED** [review]
Thank you for this great addition. While reviewing the PR we realised that we need to improve the natove token handling. I created a separate issue #219 for this.

**Pull request merged**